modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/src/main/java/controller/UserController.java

```diff
@@ -16,11 +16,19 @@
 import java.util.Calendar;
 import java.util.Locale;
 
+#if (${jakartaCompatible.equals("true")})
+import jakarta.portlet.ActionResponse;
+#else
 import javax.portlet.ActionResponse;
```

Comment:
These kind of if-else blocks should not be needed for javax/jakarta imports like this. There is a post-transform process in place via `JakartaCompatabilityUtil`.